batchflow/models/torch/efficientnet.py

```diff
@@ -0,0 +1,227 @@
+""" EfficientNet family of NNs
+
+Mingxing Tan, Quoc V. Le "`EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks
+<https://arxiv.org/abs/1905.11946>`_"
+
+"""
+
+import torch
+
+#pylint: disable=too-many-ancestors
+from .encoder_decoder import Encoder
+from .blocks import MBConvBlock
+
+
+def swish(x):
```

Comment:
I believe that this implementation is very inEfficient (pun intended)

Reply:
I'm afraid, I cannot quite understand what you mean.

Reply:
In order to compute this activation implemented this way, `Pytorch` stores a lot of intermediate activations. The usual way to avoid this is to fuse the whole operation into one by using `Torchscript` and custom forward/backward passes. Nevertheless, it has already been implemented in the `Pytorch`: [SiLU](https://pytorch.org/docs/stable/generated/torch.nn.SiLU.html), and you can just use `'common/activation' : 'silu'` in the model configuration (if the version of `torch` package is modern enough)

Reply:
`SiLU` is introduced in torch 1.7 (which is the most recent as for now). Are we going to stick to this version in batchflow?